core/cmake/modules/FindLQR-1.cmake fails to find LQR in the linking test
because the path to the library is not passed to the linker. This could
be addressed by running what pkg_check_modules() finds through find_library()
first or rewriting the module to avoid using pkgconf all together.

The local workaround is to set USES+=localbase:ldflags to avoid extra patches.

--- core/CMakeLists.txt.orig	2023-07-14 21:16:22 UTC
+++ core/CMakeLists.txt
@@ -142,13 +142,13 @@ find_package(Doxygen)
 
 find_package(Doxygen)
 
-if(NOT CMAKE_SYSTEM_NAME STREQUAL FreeBSD)
+#if(NOT CMAKE_SYSTEM_NAME STREQUAL FreeBSD)
 
     # Lqr-1 packaging and detection under FreeBSD is problematic since a very long time and break the Gitlab CI
 
     find_package(LQR-1)
 
-endif()
+#endif()
 
 if(BUILD_WITH_CCACHE)
 
